Site migration

ethanresnick opened the pull request:
@adamhooper Please review when you have a sec! I think this is the start of the right setup, though now I have to figure out whatever's involved in actually getting the blog server to start up and listen on 8000

adamhooper commented:
Looks good. But we're not going to start the blog on http://localhost:8000. It's already running on http://ip-10-20-194-108.ec2.internal:80.

ethanresnick commented:
@adamhooper Ok, I removed the commit with all the blog-related stuff now that we decided to do that outside of haproxy. If you want to merge what's left (i.e. simplifications to the `http_redirect` frontend), feel free. Otherwise, if that's more trouble than its worth, feel free to just close this + delete the branch.

ethanresnick committed:
Update hostnames for overviewdocs.com merge

These must be the _preferred_ hostnames, which haproxy will redirect to
when others are used

adamhooper commented on cloud-init/web.txt:
```diff
@@ -34,7 +34,7 @@ write_files:
         bind 0.0.0.0:80
         timeout client 50s
-        redirect prefix https://#OVERVIEW_ENVIRONMENT_ADDRESS# code 301 
+        redirect scheme https code 301
```
This isn't the same thing. It leads to double-redirect: http://overviewproject.org => https://overviewproject.org => https://www.overviewproject.org. Please undo this line.

adamhooper commented on cloud-init/web.txt:
```diff
@@ -42,10 +42,7 @@ write_files:
-        capture response header Content-Encoding len 10
-        capture response header Cache-Control len 200
-        capture response header Last-Modified len 200
+        capture response header Location len 100
```
Why this change? IIRC Logstash is expecting our capture stuff to be configured just so.
